### PR TITLE
dev-libs/device-atlas-api-c: Version bump

### PR DESCRIPTION
**Pull request by bjacquin**
In order to move Manifest to BLAKE2B as requested in bug #642874, old
version have to be removed since sources are not available anymore

Package-Manager: Portage-2.3.13, Repoman-2.3.3

### CODE REVIEW

**Comment by candrews** [comment]
Unfortunately, we can't merge this because you're removing stable versions.

You need to request version 2.1.3 to be keyworded "amd64 arm ppc x86" - then this PR can be merged. See https://wiki.gentoo.org/wiki/Stable_request

**Comment by bjacquin** [comment]
@devnexen can you please provide sources for older version for rehash ?

**Comment by devnexen** [comment]
@bjacquin Sorry I did not see this PR before but within few weeks we ll release 2.1.4 and drop 2.1.3

**Comment by bjacquin** [comment]
@devnexen well, the thing is that we need to keep at least one version as stable to make the tree consistent. One stable haproxy must depend on a stable dev-libs/device-atlas-api-c. As of today, 2.1.2_p1 is stable but we are not able to fetch the source to regenerate the Manifest hashes.

Could you post on your website the source code for 2.1.2_p1 ?

**Comment by devnexen** [comment]
I m not in control for the website and we do not keep old versions (especially in this case these are bug fixes no new features).

**Comment by bjacquin** [comment]
@candrews what would be your advise here ?

**Comment by candrews** [comment]
I suggest you update this PR to only do the version bump - leave the old versions alone. https://bugs.gentoo.org/642874 will remain open.

Then, open a stabilization request for the new version just added. Once the new version is stabilized so it has the same keywords as the old version, then you can submit a PR removing the old versions which will close https://bugs.gentoo.org/642874

**Comment by bjacquin** [comment]
Last PR is now only including the version bump

**Comment by candrews** [comment]
@mgorny I can't push this change because there's a disallowed hash in the manifest (the pre commit hook fails). Any suggestions?

**Comment by mgorny** [comment]
The proxied maintainer needs to update hashes for the old versions first.

**Comment by candrews** [comment]
@mgorny which he cannot do, as earlier comments indicate.

File: dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	eerror "Then place the file in ${DISTDIR}/${MY_P}.tgz"
+}
+
+multilib_src_install_all() {
+	if use doc; then
+		local -a HTML_DOCS=( Documentation )
+	fi
+
+	if use examples; then
+		insinto /usr/share/doc/${P}/examples
```
Comment (mgorny) [inline dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.3.ebuild]: docinto/dodoc. I'm going to fix this one for you.

**Comment by bjacquin** [comment]
@devnexen according to @mgorny comments, would you be able to find a way to make old releases available on the website ? Could you talk internally on this matter ?

File: dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.3.ebuild (hunk 1 above)
Comment (bjacquin) [reply dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.3.ebuild]: Fixed in last PR

**Comment by bjacquin** [comment]
docinto/dodoc fixed in last PR.
Patch fixed as well, this is due to rebase fixing whitespaces.

**Pull request closed without merging**